src/Raven.Server/Documents/Handlers/IndexHandler.cs

```diff
@@ -652,6 +660,19 @@ public Task Terms()
             }
         }
 
+        private string GetIndexNameFromCollectionAndField(string field, DocumentsOperationContext context)
+        {
+            var collection = GetStringQueryString("collection", false);
```

Comment:
I think that you can use  `GetQueryStringValueAndAssertIfSingleAndNotEmpty` ?

Reply:
This is logic is like this: if `name` parameter is not there, than we check for the `collection` parameter, and if it isn't there than we throw that the `name` parameter is mandatory.

Reply:
ok 💯